### PR TITLE
Create 088-ipfs-grants-platform.md

### PR DESCRIPTION
**Pull request by parkan**
Draft blog post announcing the IPFS Grants Platform. Needs to be fleshed out with more past IPFS grant examples.

### CODE REVIEW

**Comment by parkan** [comment]
@momack2 @autonome

**Comment by parkan** [comment]
one thing I'm not 100% on is whether to mention our gitcoin experiment here or in the follow-up guide, what do you think?

**Comment by momack2** [comment]
Probably the follow-up. Do you mind if I make some tweaks? I wanna separate the successes of the past quarter from the coronavirus focus for next quarter

File: content/post/088-ipfs-grants-platform.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+date: 2020-04-16
+url: 2020-04-16-ipfs-grants-platform
+title: IPFS Grants Platform
+author: Arkadiy Kukarkin
+tags: mobile-design-guidelines, Mobile, Design, Research
```
Comment (momack2) [inline content/post/088-ipfs-grants-platform.md:6]: Think this is old

Comment (parkan) [reply content/post/088-ipfs-grants-platform.md:6]: oops, good catch!

**Comment by parkan** [comment]
> 
> 
> Probably the follow-up. Do you mind if I make some tweaks? I wanna separate the successes of the past quarter from the coronavirus focus for next quarter

sure, positioning these two ideas is probably the biggest challenge, let's iterate on it

**Comment by parkan** [comment]
also @momack2 could you expand on past grants (esp PeerGos) as you have more familiarity with those?

**Comment by momack2** [comment]
Sounds good! Just took a pass at those asks. Let me know what you think!

**Comment by parkan** [comment]
ha this is actually closer to how I had it structured in a previous draft -- I had cut it down quite a lot for brevity in the pushed version

you should add yourself to the byline and I think we're good to publish

schedule for first thing Monday?

**Comment by momack2** [comment]
Done!

**Pull request merged**